extern/agg24-svn/include/agg_pixfmt_rgba.h

```diff
@@ -1521,7 +1521,7 @@ namespace agg
         };
         struct pixel_type
         {
-            value_type c[num_components];
+            value_type c[];
```

Comment:
This is failing to compile in the docs builds which is interesting, I guess we are running a different gcc version there?

Reply:
gcc requires non empty struct. clang does not care. I didn't know that. So I need to find other solution.

Reply:
Looks like there is simpler solution without UB.